[2022-04-07 19:21:56,190] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2015-03-01T00:00:00+00:00 [queued]>
[2022-04-07 19:21:56,340] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2015-03-01T00:00:00+00:00 [queued]>
[2022-04-07 19:21:56,341] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:21:56,341] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:21:56,341] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:21:57,172] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task> on 2015-03-01 00:00:00+00:00
[2022-04-07 19:21:57,206] {standard_task_runner.py:52} INFO - Started process 2625 to run task
[2022-04-07 19:21:57,249] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task', 'scheduled__2015-03-01T00:00:00+00:00', '--job-id', '12699', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpodvip43o', '--error-file', '/tmp/tmptidrzj46']
[2022-04-07 19:21:57,276] {standard_task_runner.py:80} INFO - Job 12699: Subtask create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
[2022-04-07 19:21:57,651] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2015-03-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:21:58,082] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2015-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2015-03-01T00:00:00+00:00
[2022-04-07 19:21:58,084] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:21:58,147] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:21:58,230] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2015_03_01T00_00_00_00_00_99a76babc241c7c43c59696a4b40fa40
[2022-04-07 19:21:59,078] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "creation_date" to type TIMESTAMP at [1:111]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2015_03_01T00_00_00_00_00_99a76babc241c7c43c59696a4b40fa40

[2022-04-07 19:21:59,176] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task, execution_date=20150301T000000, start_date=20220407T192156, end_date=20220407T192159
[2022-04-07 19:21:59,343] {standard_task_runner.py:98} ERROR - Failed to execute job 12699 for task create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task (400 Could not cast literal "creation_date" to type TIMESTAMP at [1:111]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2015_03_01T00_00_00_00_00_99a76babc241c7c43c59696a4b40fa40
; 2625)
[2022-04-07 19:21:59,397] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:21:59,611] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:15:20,633] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2015-03-01T00:00:00+00:00 [queued]>
[2022-04-07 22:15:20,670] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2015-03-01T00:00:00+00:00 [queued]>
[2022-04-07 22:15:20,670] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:15:20,670] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:15:20,670] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:15:20,956] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task> on 2015-03-01 00:00:00+00:00
[2022-04-07 22:15:20,972] {standard_task_runner.py:52} INFO - Started process 4888 to run task
[2022-04-07 22:15:20,995] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task', 'scheduled__2015-03-01T00:00:00+00:00', '--job-id', '13957', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpc4gcogld', '--error-file', '/tmp/tmpkvi3yc9j']
[2022-04-07 22:15:21,004] {standard_task_runner.py:80} INFO - Job 13957: Subtask create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
[2022-04-07 22:15:21,251] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2015-03-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:15:21,586] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2015-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2015-03-01T00:00:00+00:00
[2022-04-07 22:15:21,589] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:15:21,612] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:15:21,639] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2015_03_01T00_00_00_00_00_bd4b7a0cd703b16761d7414aa988e77a
[2022-04-07 22:15:25,462] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task, execution_date=20150301T000000, start_date=20220407T221520, end_date=20220407T221525
[2022-04-07 22:15:25,505] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:15:25,548] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
